string target_frame
sensor_msgs/PointCloud input_pcd
---
sensor_msgs/PointCloud transformed_pcd